plugins/webp-uploads/helper.php

```diff
@@ -471,11 +472,12 @@ function webp_uploads_get_mime_type_image( int $attachment_id, string $src, stri
 				continue;
 			}
 
-			return str_replace(
+			$result = str_replace(
 				$size_data['file'],
 				$size_data['sources'][ $mime ]['file'],
 				$src
 			);
+			return '' === $result ? null : $result;
```

Comment:
Don't forget to update the `@return` tag for `webp_uploads_get_mime_type_image()` to be `non-empty-string|null`